01_network.tf

```diff
@@ -0,0 +1,26 @@
+resource "openstack_networking_network_v2" "network_1" {
+  name           = "${var.env_name}-net"
+  admin_state_up = "true"
+}
+
+resource "openstack_networking_subnet_v2" "subnet_1" {
+  name            = "${var.env_name}-subnet"
+  network_id      = "${openstack_networking_network_v2.network_1.id}"
+  cidr            = "192.168.0.0/24"
+  ip_version      = 4
+  dns_nameservers = "${var.dns_nameservers}"
+}
+
+resource "openstack_networking_router_v2" "router_1" {
+  name                = "${var.env_name}-router"
+  external_network_id = "${data.openstack_networking_network_v2.external.id}"
+}
+
+resource "openstack_networking_router_interface_v2" "router_interface_1" {
+  router_id = "${openstack_networking_router_v2.router_1.id}"
+  subnet_id = "${openstack_networking_subnet_v2.subnet_1.id}"
+}
+
+resource "openstack_networking_floatingip_v2" "postgresql_ip" {
```

Comment:
I don't think it is necessary to give postgresql a public IP.

Reply:
Yes, you're right. I only had it there so I could temporarily use it as a bastion host to check on the MongoDB nodes, and then forgot to take it out. I'll remove this now.

Reply:
Addressed by: https://github.com/julianpistorius/terraref-terraform/commit/a2161f405021799cbbafaf8c97126729bdae3b9d